extensions/amp-access/amp-access.md

```diff
@@ -77,11 +77,7 @@ Supporting AMP Access requires that the Publisher implement the components descr
 
 To assist access services and use cases, AMP Access introduces the concept of _Reader ID_.
 
-The Reader ID is an anonymous and unique ID created by the AMP ecosystem. It is unique for each Reader/Publisher pair - a Reader is identified differently to two different Publishers. It is a non-reversible ID. The Reader ID is included in all AMP/Publisher communications and has very high entropy. Publishers can use the Reader ID to identify the Reader and map it to their own identity systems.
-
-The Reader ID is constructed on the user device and intended to be long-lived. However, it follows the normal browser storage rules, including those for incognito windows. The intended lifecycle of a Reader ID is 1 year between uses or until the user clears their cookies. The Reader IDs are not currently shared between devices.
-
-The Reader ID is constructed similarly to the mechanism used to build ExternalCID described [here](https://docs.google.com/document/d/1f7z3X2GM_ASb3ZCI_7tngglxwS6WoWi1EB3aKzdf6vo/edit#heading=h.hb9q0wpwwhuf). An example Reader ID is `amp-OFsqR4pPKynymPyMmplPNMvxSTsNQob3TnK-oE3nwVT0clORaZ1rkeEz8xej-vV6`.
+Read more about it in [`amp-subscriptions`](../amp-subscriptions/amp-subscriptions.md).
```

Comment:
If this information is important for this component, and this component can be used without `amp-subscriptions`, it should have Reader ID information included.